code/game/objects/effects/anomalies.dm

```diff
@@ -276,11 +276,14 @@
 	S.rabid = TRUE
 	S.amount_grown = SLIME_EVOLUTION_THRESHOLD
 	S.Evolve()
+	var/datum/action/innate/slime/reproduce/A = new
+	A.Grant(S)
 
-	var/list/mob/dead/observer/candidates = pollCandidatesForMob("Do you want to play as a pyroclastic anomaly slime?", ROLE_PAI, null, null, 100, S, POLL_IGNORE_PYROSLIME)
+	var/list/mob/dead/observer/candidates = pollCandidatesForMob("Do you want to play as a pyroclastic anomaly slime?", ROLE_SENTIENCE, null, null, 100, S, POLL_IGNORE_PYROSLIME)
 	if(LAZYLEN(candidates))
 		var/mob/dead/observer/chosen = pick(candidates)
 		S.key = chosen.key
+		to_chat(S, "<b>Congratulations! You are barely sentient but voracious slime with no allegiance - act as one. Think twice before sabotaging Gravity Generator, Vault, Supermatter, smashing communication consoles or attacking AI unprowoken. Those actions can ruin the round for other players and might be interpreted as a violation of <a href=https://tgstation13.org/wiki/Rules>the first rule.</a></b>")
```

Comment:
attacking AI unprowoken
attacking the AI unprovoked

Reply:
Thanks, fixed :+1: